packages/forklift-console-plugin/src/modules/Providers/ProviderRow.tsx

```diff
@@ -45,6 +46,19 @@ const toPositiveState = (conditionValue: string): 'Error' | 'Ok' | 'Unknown' =>
   }
 };
 
+const fromProviderState = (status: ProviderStatus): 'Error' | 'Ok' | 'Unknown' => {
+  switch (status) {
+    case 'Ready':
+      return 'Ok';
+    case 'ConnectionFailed':
+    case 'ValidationFailed':
+      return 'Error';
+    case 'Staging':
```

Comment:
`staging` should have an icon too

Reply:
Changed to spinner-like loading state - the same as in the standalone UI (we are re-using a legacy widget here)
![image](https://user-images.githubusercontent.com/64194103/220438124-86e260cb-d086-48bd-863c-b41e3a7f69c3.png)